setup.sh

```diff
@@ -19,16 +19,17 @@ fi
 # install plugins
 if [ -d /geogig/lib ]
```

Comment:
you should update /geogig/lib to libexec in case of dev version

Reply:
Good catch again